src/apps/config/leader.lua

```diff
@@ -631,6 +631,21 @@ function Leader:rpc_get_state (args)
    local success, response = pcall(getter)
    if success then return response else return {status=1, error=response} end
 end
+
+function Leader:rpc_get_alarms_state (args)
+   local function getter()
+      local printer = path_printer_for_schema_by_name(
```

Comment:
args.schema seems to be ignored; probably instead we should assert that args.schema is ietf-alarms, no?  Otherwise LGTM, though it needs a readme in program/alarms/.

Reply:
Right, the alarm's README.md documentation is still lacking. I plan to write it down when I have moved the other alarm related commands to this new alarms module.